src/app/features/psbt-signer/components/psbt-decoded-request/psbt-unsigned-output-list/components/psbt-unsigned-output-item.tsx

```diff
@@ -27,14 +27,14 @@ export function PsbtUnsignedOutputItem({
 
   const isOutputCurrentAddress =
     addressFromScript === addressNativeSegwit || addressFromScript === addressTaproot;
-  const outputValue = satToBtc(Number(output.amount)).toString();
+  const outputValueAsMoney = createMoney(Number(output.amount), 'BTC');
```

Comment:
```suggestion
  const outputValueAsMoney = createMoney(output.amount, 'BTC');
```

cast isn't needed right, we support creating as strings

Reply:
This is a bigint.